it/src/test/java/org/zalando/client/java/SimpleEventTest.java

```diff
@@ -58,7 +58,7 @@ public void validatePublishedNrOfEvents() throws InterruptedException, Execution
                 .withEventTypeId("SimpleEventTest-validatePublishedNrOfEvents").build();
         EventIntegrationHelper it = new EventIntegrationHelper(gen, client);
         assertTrue("EventType should be created", it.createEventType());
-        Thread.sleep(1000);// Creation can take time.
+        Thread.sleep(5000);// Creation can take time.
```

Comment:
No, not on staging, actually it was only a prolem in my local environment because of low resources on my local machine.